Lib/typing.py

```diff
@@ -1401,7 +1401,25 @@ def _is_callable_members_only(cls):
 
 
 def _no_init(self, *args, **kwargs):
```

Comment:
Please rename this to something like `_no_init_or_replace_init` (I admit my proposed name is terrible, but I couldn't think of a better one. If you think of something better, please use that instead.)

Reply:
I am thinking about something more general like `_protocol_init`

Reply:
That works too :). I'll leave it up to you.